backup/backup.go

```diff
@@ -3,11 +3,13 @@ package backup
 import (
 	"context"
 	"fmt"
+	"golang.org/x/exp/slices"
```

Comment:
In golang 1.21, slices package can be imported as "slices".
MOCO is already built with golang 1.21 #571

Reply:
[1eb294b](https://github.com/cybozu-go/moco/pull/604/commits/1eb294b96959d0c24cf853551ea0a98df289a53f)

Reply:
[0e583b0](https://github.com/cybozu-go/moco/pull/604/commits/0e583b0f6a9c8b1f800361670b0d8cb08f93658b)